functions/Get-DbaSpn.ps1

```diff
@@ -48,20 +50,15 @@ Returns a custom object with SearchTerm (ServerName) and the SPNs that were foun
     [cmdletbinding()]
 	param (
         [Parameter(Mandatory = $false,ValueFromPipeline = $true)]
-        [string[]]$ComputerName,
+        [string[]]$ComputerName = $env:COMPUTERNAME,
         [Parameter(Mandatory = $false)]
 		[string[]]$AccountName,
+		[switch]$ShowAccountNameAll,
```

Comment:
I like ByAccount, thanks! 

The parametersets would work to keep ByAccount and AccountName together - but there are definitely times when users would want to check a remote host with another AccountName. Brain isn't working, pass for now.